### DISCUSSION TITLE
Transaction Event Handler

### DISCUSSION THREAD

**Discussion by pedrofcarvalho** [mmohades/Venmo#66] [Q&A] [answered]
Hey, 
I am looking to create a way to monitor/manage the transactions from my Venmo account. Is there an event handler for transactions in this API, and if there's isn't one per-se, do you recommend a work around with any of the existing features?
I appreciate your effort!

**Comment by mmohades** [comment, marked as answer]
Sorry for the delay. I think the only way right now is to fetch the transactions periodically and monitor the changes.

**Reply by pedrofcarvalho** [reply]
Thank you!
